src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -194,18 +201,18 @@ private VCWithNewAlleles(final VariantContext vc, final List<Allele> newAlleles,
             this.isSpanningEvent = isSpanningEvent;
         }
 
-        public List<Allele> filterAllelesForFinalSet() {
+        public Stream<Allele> filterAllelesForFinalSet() {
```

Comment:
It seems weird to change this public method to return a `Stream` rather than a more conventional `List` -- and if you're only doing this to make your streaming expressions below infinitesimally more elegant then I particularly disapprove :) Does this method have to be public in the first place?